src/openapi/transport/retry.js

```diff
@@ -1,4 +1,4 @@
-﻿/**

```

Comment:
have this added BOM's? or were BOM's there before

Reply:
resolved.